content/en/blog/releases/1-9-0/index.md

```diff
@@ -0,0 +1,139 @@
+spec:
+  match:
+    any:
+    - resources:
+        kinds:
+          - Pod
+  conditions:
+    all:
+    - key: "{{ request.object.metadata.ownerReferences[] || `[]` }}"
+      operator: Equals
+      value: []
+  schedule: "0 0 * * *"
+```
+
+Here is a ClusterCleanupPolicy which, as you might have guessed, applies cluster-wide, and which will find and remove Pods with no ownerReferences every day at midnight. Cleanup policies like these can be super helpful to keep clutter to a minimum and fit nicely into existing automation workflows.
+
+A policy-based approach is only the first step, and in the second phase we intend on implementing automated cleanup based on a reserved label or annotation assigned to resources.
+
+### Distributed Tracing
+
+In the 1.8 release, we added OpenTelemetry support, but in this release we wanted to one-up ourselves and add in more observability to provide insights on _exactly_ what Kyverno is doing to your resources. So in 1.9, we've added full support for distributed tracing. With distributed tracing instrumented in Kyverno, you can see every rule and even every external call (in the case of OCI registry lookups) Kyverno is making and how long each span took. This is super helpful not just from a visibility but also a troubleshooting and even auditability standpoint. With traces in hand sent to your collector, you can see the rules which matched the resource and were processed so you know everything is working properly.
+
+![tracing](kyverno-tracing.png)
+
+### Extended Support for Subresources
+
+Kyverno has had support for some subresources for a few releases now (ephemeral containers have been supported since 1.7), but there were still gaps. Specifically, Kyverno had problems with the Scale subresource, which can be tricky to deal with, as well as mutations to the Status subresource. This meant that a couple real-world use cases that have come up simply weren't possible to implement in Kyverno policies. With Kyverno 1.9, that should be a thing of the past and most, if not all, subresources can be handled quite well.
+
+With this enhanced support, you can now easily match on the precise subresource you want (even using wildcards) and Kyverno will figure it out. For example, advertising [extended resources](https://kubernetes.io/docs/tasks/administer-cluster/extended-resource-node/) to Kubernetes nodes is important for use cases like GPUs and FPGAs, and with a Kyverno [mutate](/docs/writing-policies/mutate/) rule targeted at Node/status, it's now incredibly simple and doesn't require any custom webhooks.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: advertise-resource
+spec:
+  background: false
+  rules:
+    - name: advertise-dongle
+      match:
+        any:
+        - resources:
+            kinds:
+            - Node/status
+      mutate:
+        patchStrategicMerge:
+          status:
+            capacity:
+              example.com/dongle: 2
+```
```

Comment:
I think we should mention https://github.com/kyverno/kyverno/blob/9d2deb0568db2d23ebe4d3ea28767ae1f747a7a8/test/conformance/kuttl/mutate/clusterpolicy/standard/existing/mutate-existing-node-status/01-manifests.yaml#L1-L22 too.

This policy will **only** match incoming requests to the `status` subresource and the user may get confused about why the policy isn't working when sending requests to the `Node` resource.

Reply:
Sorry, I'm confused here. This policy above is only for "new" Nodes where the objective is only to mutate `/status` because that's what's needed. The example from the kuttl test is about existing Nodes. Can you clarify what suggestion you had in mind?

Reply:
Yeah, I just wanted to convey that the policy above won't work for updates to existing nodes (unless explicitly sending a request to the status subresource), if the intention is just for new nodes, then the policy is perfectly fine.

Reply:
Thanks, made a clarifying sentence in the blog article.